### PR TITLE
Feature - update ui detail

### PR DESCRIPTION
**Pull request by at-phatnguyen**

### CODE REVIEW

**Comment by at-daiho** [comment]
:+1: There are no linter violations.

File: MyApp/View/Controllers/Detail/DetailViewController.swift [hunk 1]
```diff
@@ -84,7 +94,33 @@ class DetailViewController: UIViewController {
 
         pageChange(currentPage: pageControl.currentPage)
     }
+
+    @IBAction func addFavoriteButtonTouchUpInside(_ sender: UIButton) {
```
Comment (at-truongnguyen) [inline MyApp/View/Controllers/Detail/DetailViewController.swift]: Change name

File: MyApp/View/Controllers/Detail/DetailViewController.swift [hunk 2]
```diff
@@ -84,7 +94,33 @@ class DetailViewController: UIViewController {
...
+        let alert = UIAlertController(title: App.Strings.alret, message: App.Strings.addToFavorite, preferredStyle: .alert)
```
Comment (at-truongnguyen) [inline MyApp/View/Controllers/Detail/DetailViewController.swift]: alret

File: MyApp/View/Controllers/Detail/DetailViewController.swift [hunk 3]
```diff
@@ -84,7 +94,33 @@ class DetailViewController: UIViewController {
...
+        let ok = UIAlertAction(title: App.Strings.ok, style: .default, handler: { [weak self] _ in
```
Comment (at-truongnguyen) [inline MyApp/View/Controllers/Detail/DetailViewController.swift]: ok?

File: MyApp/View/Controllers/Detail/DetailViewController.swift [hunk 4]
```diff
@@ -84,7 +94,33 @@ class DetailViewController: UIViewController {
...
+            guard let this = self else { return }
+            this.viewModel.addtoFavorite()
+        })
+        let cancel = UIAlertAction(title: App.Strings.cancel, style: .cancel, handler: nil)
```
Comment (at-truongnguyen) [inline MyApp/View/Controllers/Detail/DetailViewController.swift]: cancelAction

File: MyApp/View/Controllers/Detail/DetailViewController.swift [hunk 5]
```diff
@@ -84,7 +94,33 @@ class DetailViewController: UIViewController {
...
+        alert.addAction(ok)
+        alert.addAction(cancel)
+        self.present(alert, animated: true, completion: nil)
```
Comment (at-truongnguyen) [inline MyApp/View/Controllers/Detail/DetailViewController.swift]: `self`

File: MyApp/View/Controllers/Detail/DetailViewController.swift [hunk 6]
```diff
@@ -84,7 +94,33 @@ class DetailViewController: UIViewController {
...
+    }
+
+    @IBAction func addCartButtonTouchUpInside(_ sender: UIButton) {
```
Comment (at-truongnguyen) [inline MyApp/View/Controllers/Detail/DetailViewController.swift]: function name

File: MyApp/View/Controllers/Detail/DetailViewController.swift [hunk 7]
```diff
@@ -84,7 +94,33 @@ class DetailViewController: UIViewController {
...
+        let alert = UIAlertController(title: App.Strings.alret, message: App.Strings.addToCart, preferredStyle: .alert)
+        let ok = UIAlertAction(title: App.Strings.ok, style: .default, handler: { [weak self] _ in
```
Comment (at-truongnguyen) [inline MyApp/View/Controllers/Detail/DetailViewController.swift]: action

File: MyApp/View/Controllers/Detail/DetailViewController.swift [hunk 8]
```diff
@@ -84,7 +94,33 @@ class DetailViewController: UIViewController {
...
+            guard let this = self else { return }
+            this.viewModel.addtoCart()
+        })
+        let cancel = UIAlertAction(title: App.Strings.cancel, style: .cancel, handler: nil)
```
Comment (at-truongnguyen) [inline MyApp/View/Controllers/Detail/DetailViewController.swift]: action

File: MyApp/View/Controllers/Detail/DetailViewController.swift [hunk 9]
```diff
@@ -84,7 +94,33 @@ class DetailViewController: UIViewController {
...
+        alert.addAction(ok)
+        alert.addAction(cancel)
+        self.present(alert, animated: true, completion: nil)
```
Comment (at-truongnguyen) [inline MyApp/View/Controllers/Detail/DetailViewController.swift]: self

**Pull request merged**